### DISCUSSION TITLE
Support CMI for Enderchest like you do for OpenInv

### DISCUSSION THREAD

**Discussion by TomLewis** [QuickShop-Community/QuickShop-Hikari#1741] [Ideas]
Same as OpenInv, but support CMI which replaces OpenInv in all ways.
Allow use of CMI enderchests for shops.

https://www.zrips.net/cmi/api/

**Comment by creatorfromhell** [comment]
> Same as OpenInv, but support CMI which replaces OpenInv in all ways. Allow use of CMI enderchests for shops.
> 
> https://www.zrips.net/cmi/api/

Looks like all of their recent builds on their jitpack are errored out... https://jitpack.io/#Zrips/CMI-API. I'll put this on hold until they figure that out so that I have a reliable repository to pull builds from besides inserting an external jar.

**Comment by TomLewis** [comment]
Posted so they know https://github.com/Zrips/CMI/issues/9099

**Comment by TomLewis** [comment]
Could you use this? https://github.com/Zrips/CMI-API/releases

**Reply by creatorfromhell** [reply]
> Could you use this? https://github.com/Zrips/CMI-API/releases

I prefer to use maven as it's easier for contributors to build without having to go find the jars, since QS can't host them for legal purposes.
